Automattic/jetpack
# Boost: Update dashboard logo

Fixes #21926 

#### Changes proposed in this Pull Request:

* Update Jetpack Boost Dashboard with newly designed Horizontal logo

#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Run the build after checking out.
* Go to Boost dashboard.
* Verify that the logo is updated with the new design: p6TEKc-5Be-p2.
